src/main/java/school/lemon/changerequest/java/introduction/hw1/Task9.java

```diff
@@ -3,6 +3,15 @@
 public class Task9 {
 
     public static boolean isPowerOfThree(int n) {
-        return false;
-    }
+
+        if (n < 1){
+            return false;
+        }
+
+        while (n % 3 == 0) {
+            boolean b = (n / 3) == 3;
+        }
+            return true;
```

Comment:
This implementation doesn't work.
You need to re-assign value of `n/3` back to `n`, cause right now you'll have infinite loop in many cases.